base/shell/explorer/lang/nl-BE.rc

```diff
@@ -0,0 +1,200 @@
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
+
+IDM_TRAYWND MENU
+BEGIN
+    POPUP ""
+    BEGIN
+        MENUITEM SEPARATOR
+        MENUITEM "&Datum en tijd Aanpassen", ID_SHELL_CMD_ADJUST_DAT
+        MENUITEM "&Notificaties Aanpassen...", ID_SHELL_CMD_CUST_NOTIF
+        MENUITEM SEPARATOR
+        MENUITEM "Vallende Schermen", ID_SHELL_CMD_CASCADE_WND
+        MENUITEM "Schermen Horizontaal Organizeren", ID_SHELL_CMD_TILE_WND_H
+        MENUITEM "Schermen Verticaal Organizeren", ID_SHELL_CMD_TILE_WND_V
+        MENUITEM "Het Bureaublad tonen", ID_SHELL_CMD_SHOW_DESKTOP
+        MENUITEM "Ongedaan maken", ID_SHELL_CMD_UNDO_ACTION
+        MENUITEM SEPARATOR
+        MENUITEM "Taakbeheer", ID_SHELL_CMD_OPEN_TASKMGR
+        MENUITEM SEPARATOR
+        MENUITEM "De Taakbalk Vastzetten", ID_LOCKTASKBAR
+        MENUITEM "Eigenschappen", ID_SHELL_CMD_PROPERTIES
+    END
+END
+
+IDM_STARTMENU MENUEX
+BEGIN
+    POPUP ""
+    BEGIN
+        MENUITEM "", -1, MFT_SEPARATOR
+        POPUP "&Programmas", IDM_PROGRAMS
+        BEGIN
+            MENUITEM "(Leeg)", -1, MFT_STRING, MFS_GRAYED
+        END
+        POPUP "Favorieten", IDM_FAVORITES
+        BEGIN
+            MENUITEM "(Leeg)", -1, MFT_STRING, MFS_GRAYED
+        END
+        POPUP "&Documenten", IDM_DOCUMENTS
+        BEGIN
+            MENUITEM "(Leeg)", -1, MFT_STRING, MFS_GRAYED
+        END
+        POPUP "&Instellingen", IDM_SETTINGS
+        BEGIN
+            MENUITEM "&Configuratiescherm", IDM_CONTROLPANEL, MFT_STRING, MFS_GRAYED
+            MENUITEM "", -1, MFT_SEPARATOR
+            MENUITEM "&Beveiliging...", IDM_SECURITY, MFT_STRING, MFS_ENABLED
+            MENUITEM "&Netwerkverbindingen ", IDM_NETWORKCONNECTIONS, MFT_STRING, MFS_ENABLED
+            MENUITEM "&Printers en Faxes", IDM_PRINTERSANDFAXES, MFT_STRING, MFS_ENABLED
+            MENUITEM "&Taakbalk en Start Menu", IDM_TASKBARANDSTARTMENU, MFT_STRING, MFS_ENABLED
+        END
+        POPUP "Zoeken", IDM_SEARCH
+        BEGIN
+            MENUITEM "", -1, MFT_SEPARATOR
+        END
+        MENUITEM "&Help en Ondersteuning", IDM_HELPANDSUPPORT, MFT_STRING, MFS_ENABLED
+        MENUITEM "&Beheren...", IDM_RUN, MFT_STRING, MFS_ENABLED
+        MENUITEM "", 450, MFT_SEPARATOR, MFS_ENABLED
+        MENUITEM "S&ynchronizeren", IDM_SYNCHRONIZE, MFT_STRING, MFS_ENABLED
+        MENUITEM "&Uitloggen%s...", IDM_LOGOFF, MFT_STRING, MFS_ENABLED
+        MENUITEM "Verbinding beëindigen...", IDM_DISCONNECT, MFT_STRING, MFS_ENABLED
+        MENUITEM "Laptophouder losmaken", IDM_UNDOCKCOMPUTER, MFT_STRING, MFS_ENABLED
```

Comment:
`ontkoppelen` ?
(Or is that NL vs BE?)

Reply:
AFAIK you can use both, maybe ontkoppelen is better in this case